config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -313,18 +313,18 @@ ext:samenvattingPropertyGroup
   ext:summaryField
       a                 form:Field ;
       sh:order          1 ;
-      sh:path           dct:description ;
+      sh:path           ( ext:samenvattingSummaryDescription dct:description ) ;
       form:validations
                         [ a                 form:MaxLength ;
                           form:grouping     form:MatchEvery ;
                           form:max          "700" ;
                           sh:resultMessage  "Max. karakters overschreden." ;
-                          sh:path            dct:description
+                          sh:path           ( ext:samenvattingSummaryDescription dct:description )
```

Comment:
@Riadabd it is not clear why you include that extra paths. why not just 'ext:samenvattingSummaryDescription'